### PR TITLE
Feature/feature branch

### PR DESCRIPTION
**Pull request by LuisHernandezCoding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+###### Behavior
+
+- The game starts displaying the Title and ask for how many players
+- Asks for first and second player names if there are more than 1
+- The game prompts the first player to select from available moves
+- Once a player has selected a valid move, the game is updated and shows the board
+- We switch the turn to the second player and repeat behavior
+- The game ends when there is a winner or it is a draw
+
+## Table of contents
```
Comment (mricanho) [inline README.md:20]: You can delete this section

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+### Prerequisites
+
+- Ruby v 3.1.2p20
+
+### Setup
+
+- Make sure you have Ruby installed on your machine
+- Clone this repository with
+```bash
+git clone https://github.com/LuisHernandezCoding/tic_tac_toe
```
Comment (mricanho) [inline README.md:48]: This link is broken

File: lib/ui.rb [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    end
+    update_scoreboard('players')
+    configuring_player_names(@players)
+  end
+
+  # rubocop:disable Style/Semicolon
+  def configuring_player_names(players)
+    players.zero? ? (naming(1, true); naming(2, true)) : (naming(1, false); naming(2, players == 1))
+  end
+  # rubocop:enable Style/Semicolon
```
Comment (mricanho) [inline lib/ui.rb:113]: Check a way to work around this, try an if statement or other option, this is not the ruby way

File: lib/ui.rb [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    update_scoreboard("naming#{player_number}")
+  end
+
+  def getting_player_name
+    name = gets.chomp
+    while name.empty? || name.length > 20
+      @messages = ['Invalid name', 'Try again:']
+      update_ui
+      name = gets.chomp
+    end
```
Comment (mricanho) [inline lib/ui.rb:134]: This while loop creates a little bug, use a if statement instead

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi    ###Luis, 

Good job so far! 🔥🔥

Overall it's a good first attempt, the game is a little bit buggy, and some loops are creating these errors, double-check that.

Good work on the use of rubocop and the RSpec, nothing more to do there.
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mricanho** [comment]
https://www.loom.com/share/d642488a57864c5fb351bdef3981d01d

File: README.md (hunk 1 above)
Comment (LuisHernandezCoding) [reply README.md]: Thanks a lot! its deleted on commit 528723d

File: README.md (hunk 2 above)
Comment (LuisHernandezCoding) [reply README.md]: fixed now on commit 528723d

File: lib/ui.rb (hunk 3 above)
Comment (LuisHernandezCoding) [reply lib/ui.rb]: cool, changed to regular if statement instead of ternary

Comment (LuisHernandezCoding) [reply lib/ui.rb]: sorry, in commit 528723d

File: lib/ui.rb (hunk 4 above)
Comment (LuisHernandezCoding) [reply lib/ui.rb]: changed to an if statement in 528723d

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello Luis! 🔥 

You worked hard in all the instructions that I give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Pull request merged**